starboard/android/shared/media_decoder.cc

```diff
@@ -644,6 +673,73 @@ void MediaDecoder::OnMediaCodecFrameRendered(int64_t frame_timestamp) {
   frame_rendered_cb_(frame_timestamp);
 }
 
+bool MediaDecoder::Reset() {
```

Comment:
nit: I think we can now rename this function to Flush(), which I feel makes more sense.

Reply:
Done.